app/frontend/src/app/tool/page.tsx

```diff
@@ -15,19 +15,18 @@ export interface Group {
 }
 
 export default function Tool() {
-  const [llm1, setLlm1] = useState<string>("gpt2");
```

Comment:
is it not best to have them start as they are in UI, else UI should prob be blank